Tensile/Common.py

```diff
@@ -1292,6 +1292,7 @@ def getArchitectureName(gfxName):
     "MinVgprNumber":                list(range(0,256)),
 
     "MaxVgprNumber":                list(range(0,257)),
+    "MinKForGSU":                   range(1,1024)
```

Comment:
Please let us know if you checked this range in rocBLAS.

Reply:
> Please let us know if you checked this range in rocBLAS.

I agreed with Koji that MinkForGSU should be some power of 2 data instead of a range.